codes/python/chapter_array_and_linkedlist/array.py

```diff
@@ -3,23 +3,27 @@
 Created Time: 2022-11-25
 Author: Krahets (krahets@163.com)
 """
+import os.path as osp
+import random
+import sys
+import typing
 
-import sys, os.path as osp
 sys.path.append(osp.dirname(osp.dirname(osp.abspath(__file__))))
-from include import *
```

Comment:
We include all the common libs in the `include/__init__.py` file. I think it is acceptable to use `from include import *` here to simplify the codes.